setup.py

```diff
@@ -104,10 +104,13 @@ def get_package_version():
 if (os.path.exists('MANIFEST')):
     os.unlink('MANIFEST')
 
-print(f'Downloading WMO ISO XML Schemas and Codelists.xml to {USERDIR}')
+
+print('Caching schemas, codelists and topic hierarchy')
 
 if not os.path.exists(USERDIR):
     os.mkdir(USERDIR)
+
+    print(f'Downloading WMO ISO XML Schemas and Codelists.xml to {USERDIR}')
```

Comment:
The schemas and now also topic hierarchies are downloaded only if the "USERDIR" does not exist. That is a nasty catch. Wouldn't it be better to do a more fine-graned test?

Reply:
Are you saying we should force update to ensure latest?

Reply:
At least we should check (non)existence of `USERDIR/wis2-topic-hierarchy`. But perhaps the `install` command could have some additional option (e.g. `--force`) to force the re-download of all those data files. Actually, there is such an option, but it has no effect. Or is that one meant for some other purpose?

Reply:
`--force` is specific to setuptools per se and does force overwrite of the package install.  Our pywcmp artifact caching is run prior to running `setup` with custom logic.  I've pushed an update here that splits out the caching, which we can follow once WCMP2 is ready for deeper testing.